debian/control

```diff
@@ -10,25 +10,23 @@ Build-Depends: debhelper (>= 9),
                pep8,
                pkg-config,
                pyflakes,
-               python3-all,
                python3-coverage,
                python3-all-dev,
                python3-flake8,
                python3-jsonschema,
                python3-nose,
                python3-pyudev,
                python3-setuptools,
+               python3-testtools,
                python3-yaml
-Standards-Version: 3.9.5
+Standards-Version: 4.3.0
```

Comment:
Could you clarify if no changes were needed, or if some were, which ones? In d/changelog is enough.

Reply:
The only change required was the priority (although I changed that before running through https://www.debian.org/doc/packaging-manuals/upgrading-checklist.txt due to a lintian warning). I've moved that entry to be under the Standards-Version entry; does that make it clear enough?

Reply:
sure